components/html-block/html-block.js

```diff
@@ -131,14 +137,18 @@ class HtmlBlock extends LitElement {
 			:host([hidden]) {
 				display: none;
 			}
-			::slotted(*) {
+			:host([no-deferred-rendering="true"]) div.d2l-html-block-rendered {
```

Comment:
I don't really have a solid idea of why this is necessary. When this property is being reflected back to an attribute, it's being explicitly set this way, not being added/removed like I would normally expect for a boolean attribute. 😕